### PR TITLE
All the dependencies are needed at runtime too

### PR DESCRIPTION
**Pull request by halkeye**

### CODE REVIEW

**Comment by timja** [comment]
Also fixed in https://github.com/jenkins-infra/account-app/pull/144

File: Dockerfile [hunk 1]
```diff
@@ -2,7 +2,7 @@ FROM eclipse-temurin:8 AS build
 
 WORKDIR /app
 COPY . .
-RUN ./gradlew --no-daemon --info war -x test
+RUN ./gradlew --no-daemon --info war
```
Comment (timja) [inline Dockerfile:5]: I think tests should be skipped here, let's do this via CI
```suggestion
RUN ./gradlew --no-daemon --info war -x test
```

**Review by timja — APPROVED** [review]

**Comment by halkeye** [comment]
the only problem with skipping the tests in the docker image, is that the docker image will be pushed (infra.ci.jenkins.io) even if tests (ci.jenkins.io) fail

**Comment by timja** [comment]
> the only problem with skipping the tests in the docker image, is that the docker image will be pushed (infra.ci.jenkins.io) even if tests (ci.jenkins.io) fail

with https://github.com/jenkins-infra/account-app/pull/144 checks will fail on PR so shouldn't be an issue I think.

**Pull request merged**